Makefile

```diff
@@ -18,7 +18,11 @@ stop:
 clean:
 	$(MAKE) stop
 	docker-compose rm --force
+	rm -rf ./metadata/*
+	rm -rf ./keys/*
+	rm -rf ./database/*.sqlite
```

Comment:
These data are not generated using the API if you run `make run`.
I am not against adding those, but not sure if it is necessary.

Reply:
Then should we remove those lines from the API as well: 
https://github.com/vmware/repository-service-tuf-api/blob/51440aa3315b915c0e499ee91aba3a2b19a68ea0/Makefile#L38